src/WebJobs.Script.WebHost/Management/MeshInitServiceClient.cs

```diff
@@ -46,17 +52,48 @@ public Task MountBlob(string connectionString, string contentShare, string targe
         }
 
         public async Task MountFuse(string type, string filePath, string scriptPath)
-            => await Mount(new[]
+            => await SendAsync(new[]
             {
                 new KeyValuePair<string, string>("operation", type),
                 new KeyValuePair<string, string>("filePath", filePath),
                 new KeyValuePair<string, string>("targetPath", scriptPath),
             });
 
-        private async Task Mount(IEnumerable<KeyValuePair<string, string>> formData)
+        public async Task PublishContainerFunctionExecutionActivity(ContainerFunctionExecutionActivity activity)
+        {
+            try
+            {
+                if (_webHostEnvironment.InStandbyMode)
+                {
+                    _logger.LogDebug(
+                        $"Discarding function execution activity for {activity.FunctionName} in standby mode");
+                    return;
+                }
+
+                var meshInitServiceUri = _environment.GetEnvironmentVariable(EnvironmentSettingNames.MeshInitURI);
+
+                var operation = new[]
+                {
+                    new KeyValuePair<string, string>("operation", "add-fes"),
+                    new KeyValuePair<string, string>("content", JsonConvert.SerializeObject(activity)),
+                };
+                _logger.LogInformation($"Publishing function execution activity {activity} to {meshInitServiceUri}");
+
+                var response = await SendAsync(operation);
+                response.EnsureSuccessStatusCode();
+                _logger.LogInformation($"Successfully published function execution activity {activity}");
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e, $"{nameof(PublishContainerFunctionExecutionActivity)}");
+            }
+        }
+
+        private async Task<HttpResponseMessage> SendAsync(IEnumerable<KeyValuePair<string, string>> formData)
         {
             var res = await _client.PostAsync(_environment.GetEnvironmentVariable(EnvironmentSettingNames.MeshInitURI), new FormUrlEncodedContent(formData));
```

Comment:
>_environment.GetEnvironmentVariable(EnvironmentSettingNames.MeshInitURI) [](start = 46, length = 72)

Should we split this into separate endpoints now for fuse mount & fes? Before this change we were only using this url for fusemount I believe. We can make it something like meshinituri/mount and meshinituri/publishfes?

Reply:
On that note, how does the mesh init server recognize the requests currently with your change?

---
In reply to: [364031114](https://github.com/Azure/azure-functions-host/pull/5432#discussion_r364031114) [](ancestors = 364031114)

Reply:
This PR has the request handling logic ( It is based on the operation name)
https://github.com/Azure/azure-functions-docker-private/pull/44